docs/l10n.rst

```diff
@@ -329,6 +329,39 @@ the template with the context. You can use the ``ftl_files`` argument with this
     def about_view(request):
         render(request, 'about.html', {'name': 'Duder'}, ftl_files='mozorg/about')
 
+Fluent File Configuration
+-------------------------
+
+In order for a Fluent file to be extracted through automation and sent out for localization,
+it must first be configured to go through one or more distinct pipelines. This is controlled
+via a set of configuration files:
+
+- `Vendor`_, locales translated by an agency, and paid for by Marketing (locales covered by staff are also included in this group).
+- `Pontoon`_, locales translated by Mozilla contributors.
+- `Special templates`_, for locales with dedicated templates that don't go through the localization process.
+
+Each configuration file consists of a pre-defined subset of locales that bedrock supports, of which
+each group is responsible for. To add a Fluent file to a configuration, it must be included as a path
+in the chosen file. For example:
+
+.. code-block:: text
+
+    [[paths]]
+        reference = "en/mozorg/mission.ftl"
+        l10n = "{locale}/mozorg/mission.ftl"
+
+It's possible for any given Fluent file to be included in both vendor and pontoon configurations,
+in order for a page to have the broadest possible coverage. When making choices, it's important
+to consider that vendor has a cost associated with it, and pontoon leans on the good will of
+our volunteer community. Choose wisely.
```

Comment:
We should clarify this statement for marketing content (which has been very specifically defined for mozorg), as there's are only very specific options available to choose from. The config options  for this content boil down to:
1) Staff locales (2)
2) Staff locales (2) + select vendor locales (n)
3) Staff (2) + all vendor locales (18)
4) Staff + vendor + community

For anything non-marketing, it's assumed that it will always go to Pontoon.

Reply:
I’ll see if I can include expand on this, thanks 👍

Reply:
Thank you!